src/south/Modbus/Modbus.schema.jsx

```diff
@@ -53,6 +53,15 @@ schema.form = {
     defaultValue: 1,
     help: <div>Slave ID of the Modbus source</div>,
   },
+  addressOffset: {
+    type: 'OIbSelect',
+    md: 2,
+    newRow: false,
+    options: [0, -1],
```

Comment:
I don't know much about the use cases for the offset, but if it's only for Modbus / Jbus, the select should display these options instead of the integer. On the backend, depending of the value (Modbus or JBus) the right offset (0 or -1) is used. Default setting would be of course Modbus (instead of 0). What do you think ?

Reply:
I think it'll be only Modbus and JBus, you're right I'll make the changes. I also have a question, is it possible in the options field to provide key value instead of only value (Like HTML5 Select Form) ?

Reply:
Right now, it is not possible : the options are an array of values directly used as it is by the OibSelect component. That's why a small process in the class file may be used to convert JBus to "-1" and Modbus to "0"